src/components/reactTasks/dislike.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export default class Dislike extends React.Component {
+  constructor() {
+    super()
+    this.state = {disliked: true}
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+ handleClick() {
+     {this.state.disliked ? 'Dislike' : 'Disliked'}
```

Comment:
You need to use `setState` here as you have on line #11 of `comment.js`, to toggle the disliked state to either `true` or `false`